### DISCUSSION TITLE
Link to TabItem component

### DISCUSSION THREAD

**Discussion by torinouq** [withastro/starlight#1356] [Feature Requests]
### What version of `starlight` are you using?

0.15.1

### What is your idea?

make it possible to 'link' into the TabItem component, to be able to replace and define hash link "#tab-panel-???" which automatically generated and always changes

### Why is this feature necessary?

provide instructions and to direct to the Tabs page via a link, this is very necessary in the case of guidance documents that use the tabs feature

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by HiDeoo** [comment]
Thanks for the feedback.

It looks like some discussion around this request has already taken place in https://github.com/withastro/starlight/discussions/2649 so I'm going to close this discussion so we can keep all the relevant information in one place.
